### PR TITLE
i18n(zh-cn): Translate lit.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate lit.mdx

### CODE REVIEW

**Comment by liruifengv** [comment]
I also updated the Lit README: https://github.com/withastro/astro/pull/7990

File: src/content/docs/zh-cn/guides/integrations-guide/lit.mdx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+export default defineConfig({
+  // ...
+  integrations: [lit()],
+});
+```
+
+## 入门
+
+要在 Astro 中使用你的第一个 Lit 组件，你可以移步我们的 [UI 框架文档][astro-ui-frameworks]. 你将会了解到：
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/lit.mdx:66]:
```suggestion
要在 Astro 中使用你的第一个 Lit 组件，你可以移步我们的 [UI 框架文档][astro-ui-frameworks]。你将会了解到：
```

File: src/content/docs/zh-cn/guides/integrations-guide/lit.mdx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+---
+import { MyElement } from '../components/my-element.js';
+---
+
+<MyElement />
+```
+
+> 请注意，Lit 需要浏览器全局变量，例如 `HTMLElement` 和 `customElements`。因此，Lit 渲染器使用这些全局变量来模拟服务器，以便 Lit 可以运行。因此，你可能会遇到由于此原因而无法正常工作的库。
+
+### Polyfills & 水合
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/lit.mdx:99]:
```suggestion
### Polyfills 与水合
```

File: src/content/docs/zh-cn/guides/integrations-guide/lit.mdx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+## 故障排除
+
+如需帮助，请查看 [Discord](https://astro.build/chat) 上的 `#support` 频道。我们友好的支持小组成员将在此处提供帮助！
+
+你也可以查看我们的 [Astro 集成文档][astro-integration] 了解更多信息。
+
+常见问题如下：
+
+### 浏览器全局
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/lit.mdx:123]:
```suggestion
### 浏览器全局变量
```

File: src/content/docs/zh-cn/guides/integrations-guide/lit.mdx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  integrations: [vue(), lit()]
+  integrations: [lit(), vue()]
+});
+```
+
+正确的顺序可能因问题的根本原因而有所不同。但是，这并不能保证解决每个问题，而且由于这个原因，一些库不能在使用 Lit 集成时使用。
+
+### 严格的包管理器
+
+当使用像 `pnpm` 这样的[严格的包管理器](https://pnpm.io/pnpm-vs-npm#npms-flat-tree)时，你可能会遇到错误，例如 `ReferenceError: module is not defined`。要解决这个问题，请使用 `.npmrc` 文件提升 Lit 依赖项：
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/lit.mdx:147]:
```suggestion
当使用像 `pnpm` 这样的 [严格的包管理器](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) 时，你可能会遇到错误，例如 `ReferenceError: module is not defined`。要解决这个问题，请使用 `.npmrc` 文件提升 Lit 依赖项：
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**